tmc-prototype/DishMaster/DishMaster/DishMaster.py

```diff
@@ -420,232 +438,295 @@ def read_achievedPointing(self):
+        """ This method checks whether MAINTENANCE mode is allowed """
+        return self.get_state() not in [DevState.ON, DevState.ALARM, DevState.DISABLE]
         # PROTECTED REGION END #    //  DishMaster.is_SetMaintenanceMode_allowed
 
     @command(
     )
     @DebugIt()
     def SetOperateMode(self):
+        """
+        This commands triggers the Dish to transition to the OPERATE Dish Element Mode,
+        and returns to the caller. This mode fulfills the main purpose of the Dish, which is to point
+        designated directions while capturing data and transmitting it to CSP.
+        :return:
+        """
         # PROTECTED REGION ID(DishMaster.SetOperateMode) ENABLED START #
         try:
             # Command to set Dish to OPERATE Mode
             self._admin_mode = 0                        # Set adminMode to ONLINE
-            self.set_state(PyTango.DevState.ON)         # Set STATE to ON
+            self.set_state(tango.DevState.ON)         # Set STATE to ON
             self._dish_mode = 8                         # set dishMode to OPERATE
-            self.set_status("Dish is in OPERATE mode.")
-            self.devlogmsg("Dish is in OPERATE mode", 4)
-        except Exception as e:
-            print "Unexpected error in executing SetOperateMode Command on Dish", self.ReceptorNumber
-            self.devlogmsg("Unexpected error in executing SetOperateMode Command on Dish", 2)
-            print "Error message is: \n", e
-        pass
+            self.set_status(CONST.STR_DISH_OPERATE_MODE)
+            self.devlogmsg(CONST.STR_DISH_OPERATE_MODE, 4)
+        except Exception as except_occured:
+            print CONST.ERR_EXE_SET_OPERATE_MODE_CMD, self.ReceptorNumber
+            self.devlogmsg(CONST.ERR_EXE_SET_OPERATE_MODE_CMD, 2)
+            print CONST.STR_ERR_MSG, except_occured
+
         # PROTECTED REGION END #    //  DishMaster.SetOperateMode
 
     def is_SetOperateMode_allowed(self):
         # PROTECTED REGION ID(DishMaster.is_SetOperateMode_allowed) ENABLED START #
-        return self.get_state() not in [DevState.ON,DevState.OFF,DevState.FAULT,DevState.ALARM,DevState.UNKNOWN,DevState.DISABLE]
+        """ This method checks whether OPERATE mode is allowed """
+        return self.get_state() not in [DevState.ON, DevState.OFF, DevState.FAULT,
+                                        DevState.ALARM, DevState.UNKNOWN, DevState.DISABLE]
         # PROTECTED REGION END #    //  DishMaster.is_SetOperateMode_allowed
 
     @command(
-    dtype_in='str', 
-    doc_in="The timestamp indicates the time, in UTC, at which command execution should start.", 
+        dtype_in='str',
+        doc_in="The timestamp indicates the time, in UTC, at which command execution"
+               " should start.",
     )
     @DebugIt()
     def Scan(self, argin):
         # PROTECTED REGION ID(DishMaster.Scan) ENABLED START #
+        """
+        The Dish is tracking the commended pointing positions within the specified
```

Comment:
Commanded*

Reply:
Updated code documentation for CentralNode, SubarrayNode, DishLeafNode and DishMaster. Commit ID is ef9a34b.